mypy/nodes.py

```diff
@@ -249,6 +249,8 @@ class SymbolNode(Node):
 
     __slots__ = ()
 
+    __match_args__ = ("name", "fullname")
+
```

Comment:
I think "non-leaf" classes should not define this. Normally they never directly appear in an AST. There are also some examples below.

Reply:
Ok, I removed the `__match_args__` from this class. Are there any others that I should remove it from?